users/service.go

```diff
@@ -31,6 +33,12 @@ var (
 
 	// ErrPasswordFormat indicates weak password.
 	ErrPasswordFormat = errors.New("password does not meet the requirements")
+
+	// ErrAlreadyEnabledUser indicates the user is enabled.
+	ErrAlreadyEnabledUser = errors.New("the user is already enabled")
```

Comment:
Please do not use `enabled` terminology here, stick to `activated`. @dborovcanin please make sure the terminology is consistent, as I can see that there is some confusion which might lead to incosistence.

Reply:
@drasko I proposed this naming 3 days ago and @dborovcanin agreed.